docs/app/services/playground/transformers/view-engine-downgrade.ts

```diff
@@ -26,7 +26,6 @@ function isViewEngineModule(playgroundModule: IPlaygroundModule): boolean {
         ? playgroundModule.imports
         : [playgroundModule.imports];
     return (
-        playgroundModule.library === 'angular-split' ||
         playgroundModule.library === 'ng2-file-upload' ||
         imports.findIndex(_import => _import === 'MarqueeWizardModule') >= 0
```

Comment:
Would it be possible to update ng2-file-upload as well if it is a view engine library? I think that should be fairly trivial as its just an example, not a dependency or peer dependency of our library.

And I assume MarqueeWizardModule is only here because it relies on angular split?

If we are able to update ng2-file-upload then I believe we can remove the whole view engine transformer all together.